src/components/Professors/ProfessorsTable.tsx

```diff
@@ -0,0 +1,55 @@
+import { ChevronRightIcon } from "@chakra-ui/icons";
+import { Button } from "@chakra-ui/react";
+import { useMemo } from "react";
+import C2Table from "../C2Table";
+import { CompleteStatusBadge } from "../CompleteStatusBadge";
+
+const ProfessorsTable = ({ professors, openDetails }) => {
+    return (
+        <C2Table
```

Comment:
Is there a way to make the filter inputs more explicit for which column they are filtering? Like a placeholder maybe?

Reply:
Do you mean in the filtering dialog boxes?

Reply:
Yeah these ones
<img width="1148" alt="image" src="https://user-images.githubusercontent.com/38937780/173687068-e75a319c-e276-4007-80b1-20a5f50e00fe.png">

Maybe even
a dropdown for the type etc...

I made a proposal branch here
https://github.com/seng499-company2/front-end/tree/57-table-filter-proposal